mlflow/gateway/client.py

```diff
@@ -79,6 +79,7 @@ def _call_endpoint(self, method: str, route: str, json_body: Optional[str] = Non
         augmented_raise_for_status(response)
         return response
 
+    @experimental
```

Comment:
Can we mark fluent APIs experimental too?

Reply:
they already were :)

Reply:
The client class was marked as experimental only before (which can be misleading when looking at the docs)